frontend/app-development/features/overview/components/Navigation/Navigation.tsx

```diff
@@ -2,17 +2,16 @@ import React from 'react';
 import classes from './Navigation.module.css';
 import { useTranslation } from 'react-i18next';
 import { Heading, Tag } from '@digdir/designsystemet-react';
-
-import { getFilteredTopBarMenu } from 'app-development/layout/AppBar/appBarConfig';
+import { getFilteredTopBarMenu } from 'app-development/utils/headerMenu/headerMenuUtils';
 import { Link } from 'react-router-dom';
-import { TopBarMenu } from 'app-shared/enums/TopBarMenu';
+import { HeaderMenuItemKey } from 'app-development/enums/HeaderMenuItemKey';
 import { RepositoryType } from 'app-shared/types/global';
 
 export const Navigation = () => {
   const { t } = useTranslation();
 
   const menuItems = getFilteredTopBarMenu(RepositoryType.App).filter(
-    (item) => item.key !== TopBarMenu.About,
+    (item) => item.key !== HeaderMenuItemKey.About && item.key !== HeaderMenuItemKey.Deploy,
```

Comment:
This filter seems to be done in many places, and it looks the have the same reason to change cross files as well. Let's see if we can make them more DRY. 😊

Reply:
See comment above 😄